From: Michael Tokarev <mjt@tls.msk.ru>
Subject: readlink is in /bin not /usr/bin on debian
Date: Fri, 14 Nov 2014 19:11:51 +0300
Bug-Debian: http://bugs.debian.org/766416
Forwarded: no

This is a debian-specific change, upstream ships
the rule to use /usr/bin/readlink while on debian
it is /bin/readlink

--- a/udev-md-raid-arrays.rules
+++ b/udev-md-raid-arrays.rules
@@ -35,7 +35,7 @@
 ENV{MD_LEVEL}=="raid[1-9]*", ENV{SYSTEMD_WANTS}+="mdmonitor.service"
 
 # Tell systemd to run mdmon for our container, if we need it.
-ENV{MD_LEVEL}=="raid[1-9]*", ENV{MD_CONTAINER}=="?*", PROGRAM="/usr/bin/readlink $env{MD_CONTAINER}", ENV{MD_MON_THIS}="%c"
+ENV{MD_LEVEL}=="raid[1-9]*", ENV{MD_CONTAINER}=="?*", PROGRAM="/bin/readlink $env{MD_CONTAINER}", ENV{MD_MON_THIS}="%c"
 ENV{MD_MON_THIS}=="?*", PROGRAM="/usr/bin/basename $env{MD_MON_THIS}", ENV{SYSTEMD_WANTS}+="mdmon@%c.service"
 
 LABEL="md_end"
